doc/man3/BIO_s_mem.pod

```diff
@@ -114,6 +114,19 @@ preceding that write operation cannot be undone.
 Calling BIO_get_mem_ptr() prior to a BIO_reset() call with
 BIO_FLAGS_NONCLEAR_RST set has the same effect as a write operation.
 
+Calling this function implies claiming ownership of the data pointer in
+the BIO.  Regardless of the state of BIO_set_close(), it is the responsibility of
+the caller to free the returned pointer to prevent leaks
```

Comment:
This change really really cannot be done. It is an undetectable API break. It would be much better to document properly the current behavior with BIO_NOCLOSE.